setup.py

```diff
@@ -147,6 +155,7 @@ def add_from_flags(envname, flag_key, dirs):
     add_from_path("CPATH", default_header_dirs)
     add_from_path("C_INCLUDE_PATH", default_header_dirs)
     add_from_flags("CPPFLAGS", "-I", default_header_dirs)
+    add_from_flags("CFLAGS", "-I", default_header_dirs)
```

Comment:
the logic of `setup.py` is completely obscure to me. Who reads `CFLAGS` from the environment? line 158 or 433 below?